router/local_peer.go

```diff
@@ -252,7 +254,7 @@ func (peer *LocalPeer) handleConnectionEstablished(conn *LocalConnection) {
 	peer.Lock()
 	peer.version += 1
 	peer.Unlock()
-	log.Println("Peer", peer.Name, "established active connection to remote peer", conn.Remote().Name, "at", conn.RemoteTCPAddr())
+	conn.log("connection fully established")
```

Comment:
We've lost the logging of the remote IP address here, which could be of interest.  Is the reasoning that we will have logged it earlier in Router.acceptTCP() ?